.github/workflows/unit-tests.yaml

```diff
@@ -46,6 +46,10 @@ jobs:
           fetch-depth: 1
       - name: Build binaries
         run: ./scripts/build.sh
+      - name: Print binaries size
+        run: ls -lh ./bin
+      - name: Check binary size
+        run: ./scripts/check-size.sh ./bin/*linux-x86_64
```

Comment:
Why is it only linux binaries?

Reply:
Different architecture have different sizes. There is only on configured limit that I chose based on linux_amd64 build.
For example, darwin_arm already is bigger than this limit, we should keep only one as reference.
This check does not prevent us from increasing the binary size but you will need to increase the limit in the culprit PR so it will be clear when adding a feature that the CLI increase in size.